src/highlighted-card.tsx

```diff
@@ -158,15 +159,17 @@ const Content: React.FC<Props> = (props) => {
     }
 
     return content;
-};
+});
 
-const HighlightedCard: React.FC<Props> = ({'aria-label': ariaLabel, ...props}) => {
-    const label = ariaLabel ?? props.title;
-    return (
-        <MaybeDismissable onClose={props.onClose} aria-label={label} width={props.width}>
-            <Content {...props} aria-label={label} />
-        </MaybeDismissable>
-    );
-};
+const HighlightedCard = React.forwardRef<HTMLDivElement, Props>(
+    ({'aria-label': ariaLabel, ...props}, ref) => {
+        const label = ariaLabel ?? props.title;
+        return (
+            <MaybeDismissable onClose={props.onClose} aria-label={label} width={props.width}>
+                <Content {...props} aria-label={label} ref={ref} />
```

Comment:
Note this. I'm forwarding the ref to the `Content`, which is the card, not to the `MaybeDismissable` wrapper

Reply:
ok, makes sense